ref/appliance-command-reference/init.md

```diff
@@ -0,0 +1,20 @@
+The `init` command configures and initialises the Anbox Cloud Appliance. This command runs through the initial bootstrap process before Anbox Cloud can be used. It provides an interactive configuration dialog which allows you to either accept the defaults or specify custom preferences for configurable aspects of the appliance. 
+
+See [Initialise the appliance](https://discourse.ubuntu.com/t/22681#initialise) for more information.
+
+## Usage
+
+    anbox-cloud-appliance init <options>
+
+## Options
+
+|Option | Description |
+|-------|-------------|
+|`--auto`|To enable automatic (non-interactive) mode. Use this option if you want to accept the default options for the appliance.|
+|`--charm-channel <channel>`|To specify the store channel from which Anbox Cloud charms are to be deployed|
+|`--force-network-bridge`|To force using a local network bridge instead of a FAN overlay network|
+|`--image-server-auth <credentials>`|To specify authentication credentials for the Anbox Cloud image server|
+|`--image-server-url <url>`| To specify the URL of the Anbox Cloud image server|
```

Comment:
Also I am 100% sure if we want to list all options here for some of them like `--image-server-ath` and `--image-server-url`, they're mainly used by us/CI for testing

Reply:
If it is publicly available for the user to see and use, we should list it. Else, we can drop it. WDYT?

Reply:
We should document them if they show up in `--help`

Reply:
Maybe we should have an "advanced options" section then? Or maybe add "(Advanced)" or something similar in the description.

That way, users know which options to look at first, before trying random options if things don't work the way they want.

Reply:
What are the advanced options for the commands that we have? AFAICS, only the `init` has a lot of options. Others have 1-3 options at max.

Reply:
Sorry, let me rephrase: `--image-server-path`, `--image-server-url`, probably also `--charm-channel` are options which we mostly use for development purposes. Even if they are publicly available, most, if not all, users should never need them. It may be better in that case to be very explicit that these are not useful for regular use cases.